platform-hub-api/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ruby:2.3.3
```

Comment:
We do, but it makes too many assumptions (via it's `ONBUILD` steps) so it's unusable for this case. We should do a "plainer" image at some point, but just to unlock this work I'm using the official Ruby image. We can certainly revisit this before we deploy to prod.